streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -5581,7 +5581,7 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
     def fail[E](e: E)(implicit trace: Trace): IO[Option[E], Nothing] = ZIO.fail(Some(e))
     def failCause[E](c: Cause[E])(implicit trace: Trace): IO[Option[E], Nothing] =
       Exit.failCause(c).mapError(Some(_))
-    def empty[A](implicit trace: Trace): IO[Nothing, Chunk[A]]   = ZIO.succeed(Chunk.empty)
+    def empty[A](implicit trace: Trace): IO[Nothing, Chunk[A]]   = Exit.emptyChunk
```

Comment:
I think this change is fine since the method is not public. Also I don't think it's used anywhere 🤔

Reply:
Indeed it's never used. Do we want to remove it?